pkg/skaffold/jib/jib.go

```diff
@@ -78,3 +100,33 @@ func getDependencies(cmd *exec.Cmd) ([]string, error) {
 	sort.Strings(deps)
 	return deps, nil
 }
+
+// calculateRoots returns a list of possible symlink-expanded paths
+func calculateRoots(path string) ([]string, error) {
+	path, err := filepath.Abs(path)
+	if err != nil {
+		return nil, errors.Wrapf(err, "unable to resolve %s", path)
+	}
+	canonical, err := filepath.EvalSymlinks(path)
```

Comment:
based on my understanding reading `EvalSymlinks`'s godoc, it will leave relative paths as relative. This should be fine right?